azure/templates/pds-tests-int.yml

```diff
@@ -42,7 +42,6 @@ steps:
       export CLIENT_ID="$(CLIENT_ID_INT)"
       export CLIENT_SECRET="$(CLIENT_SECRET_INT)"
       export TEST_PATIENT_ID="$(TEST_PATIENT_ID)"
-      export ID_TOKEN_NHS_LOGIN_PRIVATE_KEY_ABSOLUTE_PATH="$(Pipeline.Workspace)/secrets/$(ID_TOKEN_NHS_LOGIN_PRIVATE_KEY)"
```

Comment:
This command was causing a  benign 'command not found' output during the INT stage, Run Smoke Tests step of the Release pipeline. It looks to be a copy-pasta of the `pds-test.yml`, but the corresponding secrets call  was not pasted across. 

Given the command fails to run, and things still run and pass, I have removed this redundant line.